go.mod

```diff
@@ -6,7 +6,7 @@ require (
 	github.com/gliderlabs/ssh v0.3.5
 	github.com/hugelgupf/p9 v0.2.1-0.20230814004337-e6037077d6dc
 	github.com/kevinburke/ssh_config v1.2.0
-	github.com/u-root/u-root v0.11.1-0.20230913033713-004977728a9d
+	github.com/u-root/u-root v1.0.1
```

Comment:
Is this intentional? I thought we used Go-style versioning where it's risky to go above v0.x.